src/components/v2/Layout/Sidebar/styles.ts

```diff
@@ -47,31 +63,62 @@ export const useStyles = ({ expanded }: { expanded: boolean }) => {
     listItem: css`
       transition: color 0.3s;
       color: ${theme.palette.text.secondary};
-      min-width: auto;
+
       padding: 0;
       a {
-        padding-left: ${theme.spacing(2)};
-        padding-right: ${theme.spacing(0.5)};
-        padding-top: ${theme.spacing(3)};
-        padding-bottom: ${theme.spacing(3)};
+        padding: ${theme.spacing(2)} ${theme.spacing(2)} ${theme.spacing(2)} ${theme.spacing(4)};
         display: inline-flex;
-        justify-content: space-between;
+        justify-content: start;
+        ${theme.breakpoints.down('lg')} {
+          padding: ${theme.spacing(2)};
+          justify-content: center;
+        }
+        width: 100%;
+      }
+      ${theme.breakpoints.down('lg')} {
+        :hover {
+          border-radius: ${theme.shape.borderRadius.large}px;
```

Comment:
The border radius used in the design is `16px` which we currently don't have in the theme.
Perhaps we could add a `medium` size that corresponds to a 16 pixel border-radius?